scripts/ci/30_test.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -euxo pipefail
 
+: "${DD_CIVISIBILITY_AGENTLESS_ENABLED:=true}"
+: "${DD_SITE:=datadoghq.eu}"
+: "${DD_ENV:=dev}"
+: "${DD_SERVICE:=unit-test-action}"
+: "${DD_API_KEY:='<set-api-key>'}"
+: "${NODE_OPTIONS:=--max-old-space-size=8192}"
+
+# Default to big old-space, and more options for testing, but allow overriding
+NODE_OPTIONS="--max-old-space-size=8193 --unhandled-rejections=warn --require=dd-trace/ci/init ${NODE_OPTIONS:-}"
```

Comment:
Could this not cause errors if someone else defines this and now it is defined twice?

Reply:
Later flags should override earlier flags if defined multiple times (if boolean or single value), or add to a list (like with `--require`, which would add _more_ `--require`s)